### PR TITLE
Update tool_dependencies.xml

### PR DESCRIPTION
**Pull request by bgruening**
Old URL disappeared :(

### CODE REVIEW

File: packages/package_pcre_8_34/tool_dependencies.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <package name="pcre" version="8.34">
         <install version="1.0">
             <actions>
-                <action type="download_by_url">ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/pcre-8.34.tar.gz</action>
+                <action type="download_by_url">http://ftp.exim.llorien.org/pcre/pcre-8.34.tar.bz2</action>
```
Comment (nsoranzo) [inline packages/package_pcre_8_34/tool_dependencies.xml]: What about http://downloads.sourceforge.net/project/pcre/pcre/8.34/pcre-8.34.tar.bz2 ?

**Comment by bgruening** [comment]
As you like, the exim project is mirroring it and saving all versions as well.

**Comment by nsoranzo** [comment]
Sourceforge seems to be more "official" since it is also linked in [pcre homepage](http://www.pcre.org/).

**Comment by bgruening** [comment]
Updated.

**Pull request merged**